test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -154,6 +154,19 @@ def test_vol(self, gain, gain_type):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
 
+    @parameterized.expand([
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': True, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': True}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': True, 'norm_vars': True}, ),
+        ({'cmn_window': 500, 'min_cmn_window': 50, 'center': False, 'norm_vars': False}, ),
+    ])
+    def test_sliding_window_cmn(self, kwargs):
+        sample_rate = 8000
+        transform = T.SlidingWindowCmn(**kwargs)
+        waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
```

Comment:
The input to `SlidingWindowCmn` is supposed to be spectrogram.
This has been fixed in the master documentation https://pytorch.org/audio/master/functional.html#torchaudio.functional.sliding_window_cmn

Can you use `get_spectrogram`, then flip the last axis so that Tensor dimension is `[... time, freq]`?

Reply:
From the [doc](https://pytorch.org/docs/stable/generated/torch.stft.html)  of `torch.stft` I could find that it returns a tensor in the shape  (* × N × T) so do you suggest using `torch.transpose(-2, -1)` on the output?